App.tsx

```diff
@@ -246,11 +261,21 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                         <InfoListItem
                             dense
                             IconClass={Notifications}
-                            iconColor={PXBColors.red[500]}
+                            iconColor={theme.dark ? PXBColors.red[200] : PXBColors.red[500]}
                             title={'1 Alarm'}
                         />
-                        <InfoListItem dense IconClass={Info} iconColor={PXBColors.blue[500]} title={'1 Event'} />
-                        <InfoListItem dense IconClass={Clouds} title={'Online'} />
+                        <InfoListItem
+                            dense
+                            IconClass={Info}
+                            iconColor={theme.dark ? PXBColors.blue[200] : PXBColors.blue[500]}
+                            title={'1 Event'}
+                        />
+                        <InfoListItem
+                            dense
+                            IconClass={Clouds}
+                            title={'Online'}
+                            iconColor={theme.dark ? PXBColors.black[200] : theme.colors.text}
```

Comment:
default icon color on the ILI should be smart enough to handle this...i'm curious what's going on over there.

Reply:
Same issue as explained above. I'm thinking the theme isn't being updated in that useCallback function.